src/inject/dynamic-theme/inline-style.ts

```diff
@@ -317,6 +319,11 @@ export function overrideInlineStyle(element: HTMLElement, theme: FilterConfig, i
         }
         if (overrides.hasOwnProperty(property)) {
             setCustomProp(property, property, value);
+        } else {
+            const customProp = customPropsList[property];
+            if (customProp && (!element.style[customProp] && !element.hasAttribute(customProp))) {
```

Comment:
`style` property acts a little bit different, e.g. `style.backgroundColor` is used to get `background-color`. So you can use `style.getPropertyValue`. Also, what is `hasAttribute` meant to do?

Reply:
> Also, what is hasAttribute meant to do?

The original value can be either in either in a attribute(e.g. `fill="red"``) or as style(`style="fill: red"`)